docs/competency-api/documentation.md

```diff
@@ -5,4 +5,94 @@ nav_order: 1
 parent: Condition
 grand_parent: Competency API
 ---
-# Documentation
\ No newline at end of file
+# Documentation
+
+## Description
+
+### Headers for all requests
+```json
+{
+    Bearer: "Bearer Example.bearer.token"
+}
+```
+
+### Create
+
+#### post:
```

Comment:
Does create not have a body? I would add a description and examples of how to do these.

Reply:
Can we add curl request examples of how to make each of these requests also.

Reply:
It has the same body as update, I'll make it more clear that they are the same

Reply:
For the curl request part, postman allows you to generate a network request in different languages.